src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -394,6 +394,8 @@ ttyComponent = TTY
 # io/Joystick.java
 #
 ioBitWidthAttr = Bit Width
+joystickCoordinateX = Output: x-coordinate
```

Comment:
@davidhutchens Do `x-coordinate`/`y-coordinate` (vs `X coordinate`/`Y coordinate` etc) look correct in English?

Reply:
x-coordinate is the usual way to write it in English; but "X coordinate" is also acceptable. I just did a search and even found a page with the title "X coordinate - Math" and the first sentence begins "The x-coordinate is ..."